controlplane/kubeadm/main.go

```diff
@@ -341,9 +342,24 @@ func setupReconcilers(ctx context.Context, mgr ctrl.Manager) {
 		os.Exit(1)
 	}
 
+	req1, _ := labels.NewRequirement("tier", selection.Equals, []string{"control-plane"})
+	req2, _ := labels.NewRequirement("component", selection.In, []string{"kube-apiserver", "kube-controller-manager", "kube-scheduler", "etcd"})
+	podSelector := labels.NewSelector().Add(*req1, *req2)
```

Comment:
We did elsewhere:
```suggestion
	must := func(r *labels.Requirement, err error) labels.Requirement {
		if err != nil {
			panic(err)
		}
		return *r
	}

	podSelector := labels.NewSelector().Add(
		must(labels.NewRequirement("tier", selection.Equals, []string{"control-plane"})),
		must(labels.NewRequirement("component", selection.In, []string{"kube-apiserver", "kube-controller-manager", "kube-scheduler", "etcd"})),
	)
```

So it also ensures on changes we keep it right :-)

Reply:
Fixed,
But we have still 4 places where we are using "req, _ := labels.NewRequirement", I will follow up